pkg/customizations/oscap/oscap.go

```diff
@@ -66,6 +66,18 @@ func NewTailoringConfig(tc *blueprint.OpenSCAPTailoringCustomizations, r *Remedi
 		return nil, nil
 	}
 
+	if tc.XMLFilepath != nil {
```

Comment:
Should we also check that the `*tc.XMLFilepath` isn't `""`, or is that not possible?

Reply:
Yeah flip sorry I missed that. Good catch